src/azure/storage/config.rs

```diff
@@ -44,4 +44,22 @@ pub struct Config {
     ///
     /// This is part of use AAD(Azure Active Directory) authenticate on Azure VM
     pub endpoint: Option<String>,
+    /// `azure_federated_token` value will be loaded from:
+    ///
+    /// - this field if it's `is_some`
+    /// - env value: [`AZURE_FEDERATED_TOKEN`]
+    /// - profile config: `azure_federated_token_file`
+    pub azure_federated_token: Option<String>,
+    /// `azure_federated_token_file` value will be loaded from:
+    ///
+    /// - this field if it's `is_some`
+    /// - env value: [`AZURE_FEDERATED_TOKEN_FILE`]
+    /// - profile config: `azure_federated_token_file`
+    pub azure_federated_token_file: Option<String>,
+    /// `azure_tenant_id_env_key` value will be loaded from:
+    ///
+    /// - this field if it's `is_some`
+    /// - env value: [`AZURE_TENANT_ID_ENV_KEY`]
+    /// - profile config: `azure_tenant_id_env_key`
+    pub azure_tenant_id_env_key: Option<String>,
```

Comment:
This env is for `tenant_id`

Reply:
Yes, is this required? Just saw that used in [WorkloadIdentityCredential](https://github.com/Azure/azure-sdk-for-rust/blob/22ab1128f183fc80b97bb53b15486cc60702b61c/sdk/identity/src/token_credentials/workload_identity_credentials.rs#L27) in azure sdk.

Reply:
Not required, but for improve reading. Please use `tenant_id` and env `AZURE_TENANT_ID`